Library/Formula/arx.rb

```diff
@@ -0,0 +1,24 @@
+class Arx < Formula
+  desc "bundles files and programs for easy transfer and repeatable execution"
+  homepage "https://github.com/solidsnack/arx"
+  url "https://github.com/solidsnack/arx/archive/0.2.1.tar.gz"
+  sha256 "c3e91e4b9f72353061b0a104ee752da88770d695ef6329eca0f1049a56e9c110"
+
+  depends_on "ghc" => :build
+  depends_on "cabal-install" => :build
+
+  def install
+    system "cabal", "sandbox", "init"
+    system "cabal", "update"
+    system "cabal", "install", "--only-dependencies"
+    system "make"
+    bin.install "tmp/dist/arx-0.2.1-amd64-osx/arx" => "arx"
+  end
+
+  test do
+    arx_output = `arx tmpx // echo 'testing'`
+    File.write("testing.sh", arx_output)
```

Comment:
Should be able to do `(testpath/"testing.sh").write arx_output` here rather than throwing out to `File` directly, I believe.